src/bindings/python/flux/core/handle.py

```diff
@@ -173,3 +178,44 @@ def get_rank(self):
 
     def attr_get(self, attr_name):
         return self.flux_attr_get(attr_name).decode("utf-8")
+
+    def reactor_run(self, reactor=None, flags=0):
+        """
+        Run reactor associated with this Flux handle or reactor argument
+        if it is provided. Sets a signal watcher for SIGINT to return
+        from the reactor on Ctrl-C, and raise KeyboardInterrupt.
+        """
+        import signal
+
+        if not reactor:
```

Comment:
Totally feel free to leave this as is since it'll work the way it's written.  FWIW, when testing if something is `None`, I normally go for `if X is None`.  `if not X` will also technically match on `False` and an empty list.  Not really an issue here, just figured I'd mention it.

Reply:
Sounds good, thanks